src/app/components/MediaLoader/configs/aresMedia.ts

```diff
@@ -107,7 +115,8 @@ export default ({
       ...(pageType === 'mediaArticle' && { preload: 'high' }),
       statsObject: {
         ...basePlayerConfig.statsObject,
-        clipPID: versionID,
+        clipPID: subType === 'clip' ? videoId : null,
+        episodePID: subType === 'episode' ? videoId : null,
```

Comment:
Following the pattern in Morph where we provide null for the non-applicable value.

Reply:
Looking at the SMP/ Cookbook docs I see that this should use the pid and not the vpid (version pid).

I have updated this here and in tests.